modules/core/history.js

```diff
@@ -577,18 +577,27 @@ export function coreHistory(context) {
 
 
         // is iD not open in another window and it detects that
-        // there's a history stored in localStorage that's recoverable?
+        // there's a history stored in async storage that's recoverable?
         restorableChanges: function() {
-            return lock.locked() && !!context.storage(getKey('saved_history'));
+            var isLocked = lock.locked();
+
+            if (isLocked) {
+                return Promise.resolve(true);
```

Comment:
I'm pretty sure native Promises don't work in IE11.. Can you use localForage's callback API instead?

Reply:
Oh, my fault. Rewrited to callback API.